README.md

```diff
@@ -247,6 +248,20 @@ constructorIO.trackRecommendationResultsView(podID: "pdp_best_sellers", numResul
 constructorIO.trackRecommendationResultClick(podID: "pdp_best_sellers", strategyID: "best_sellers", customerID: "P183021", variationID: "7281930", numResultsPerPage: 30, resultPage: 1, resultCount: 15, resultPositionOnPage: 1, resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
 ```
 
+### Quiz Events
+
+```swift
+// Track when a quiz result is clicked
+ConstructorIo.trackQuizResultClick(quizID: "coffee-quiz", quizVersionID: "1231244", quizSessionID: "123", customerID: "123", itemName: 
+"espresso")
+
+// Track when quiz results are loaded
+ConstructorIo.trackQuizResultsLoaded(quizId: "coffee-quiz", quizVersionId: "1231244", quizSessionId: "123", resultCount: 20)
```

Comment:
Small typo here:
* `quizId` => `quizID`
* `quizVersionId` => `quizVersionID`
* `quizSessionId` => `quizSessionID`

Reply:
ah the woes of copy-pasta. Looks like the examples in the code file itself are wrong too. I'll update them.